react-app/DEVELOPMENT.md

```diff
@@ -1,16 +1,29 @@
 
 ## Running the development server
 ```
-npm run start
+API_HOST=http://localhost:4502 npm run start
 ```
-This will open the local dev server on port 9000. You can navigate then to http://localhost:9000/content/we-retail-journal/react/home.html
-
+This will open the local dev server on port 9000. You can navigate then to `http://localhost:9000/content/we-retail-journal/react/home.html`
+where `API_HOST` points to your AEM instance.
 ### GOTCHA's - you should read this
 
 #### Your requests to AEM are failing
-Most probably this is due to CORS rejection, so you might want to either config AEM to accept CORS or bypass it from the browser if you are doing local development.  
+Most probably this is due to CORS rejection, so you might want to either config AEM to accept CORS or bypass it from the browser if you are doing local development. 
```

Comment:
`config` > `configure`

Reply:
angular-app/DEVELOPMENT.md needs this updated too

Reply:
Yes that is true, at least with these common setup steps